opendp/smartnoise/core/value.py

```diff
@@ -104,6 +104,7 @@ def serialize_release(release_values):
         values={
             component_id: serialize_release_node(release_node)
             for component_id, release_node in release_values.items()
+            if release_node['value'] is not None
```

Comment:
It seems like we have the potential to return an empty list here. Is that an eventuality we want for a serialize function? Or should we be raising a warning? an error?

Reply:
Yes, you can have an empty release. This is fine.